site/docs/transform/joinaggregate.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+menu: docs
+title: Join Aggregate
+permalink: /docs/joinaggregate.html
+---
+
+The joinaggregate transform extends the input data objects with aggregate values. Aggregation is performed and the results are then joined with the input data. This transform can be helpful for creating derived values that combine both raw data and aggregate calculations, such as percentages of group totals. This transform is a special case of the [window](window.html) transform where the `frame` is always `[null, null]`.
```

Comment:
> The joinaggregate transform extends the input data objects with aggregate values.

1) Explicitly mention that the extension is a new column?  (Right now it's still a bit ambiguous.)


> This transform is a special case of the [window](window.html) transform where the `frame` is always `[null, null]`.

It's good to compare this to `window`, but perhaps it's more important to highlight the difference between this and normal aggregate.  (Similarly, add docs to aggregate to mention that "use joinaggregate if you want to preserve the table structure and instead add a new column for the aggregated values.

Reply:
Done